modin/pandas/general.py

```diff
@@ -271,6 +271,25 @@ def to_numeric(arg, errors="raise", downcast=None):  # noqa: PR01, RT01, D200
     return arg._to_numeric(errors=errors, downcast=downcast)
 
 
+@_inherit_docstrings(pandas.qcut, apilink="pandas.qcut")
+@enable_logging
+def qcut(
+    x, q, labels=None, retbins=False, precision=3, duplicates="raise"
+):  # noqa: PR01, RT01, D200
+    """
+    Quantile-based discretization function.
+    """
+    kwargs = {
+        "labels": labels,
+        "retbins": retbins,
+        "precision": precision,
+        "duplicates": duplicates,
+    }
+    if not isinstance(x, Series):
```

Comment:
Do not see a point to test pandas against pandas.

Reply:
It's not 'pandas' against 'pandas', it's 'pandas' against 'logic to fallback to pandas + pandas'. You may have potentially forgotten to pass some of the parameters or passed an extra parameter thus causing a 'got an unexpected argument' error. I believe it's not the case here but it's a good habit to test cases like this (someone in the future may break this branch accidentally and non-fully coverable tests would silently allow this to be merged).

Reply:
> You may have potentially forgotten to pass some of the parameters or passed an extra parameter thus causing a 'got an unexpected argument' error.

The error will be the same in both pandas and Modin, won't it?

Reply:
Imagine if we add our own implementation in the future and, for example, accidentally return the wrong type. This will not significantly slow down our testing, but it will save us from errors in the future.

Reply:
I would say when we add a new implementation, we will add tests then. Why do we need to bloat up tests now, which verify the results of pandas against pandas?

Reply:
I assume that your intention to add a test is just to cover codecov but maybe I am wrong.

Reply:
Added a test case for this branch.

Reply:
There is always the possibility that this can be missed during the review.